yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -239,23 +236,22 @@ private[spark] class Client(
     /**
      * Copy the given main resource to the distributed cache if the scheme is not "local".
      * Otherwise, set the corresponding key in our SparkConf to handle it downstream.
-     * Each resource is represented by a 4-tuple of:
+     * Each resource is represented by a 3-tuple of:
      *   (1) destination resource name,
      *   (2) local path to the resource,
      *   (3) Spark property key to set if the scheme is not local, and
-     *   (4) whether to set permissions for this resource
      */
     List(
-      (SPARK_JAR, sparkJar(sparkConf), CONF_SPARK_JAR, false),
-      (APP_JAR, args.userJar, CONF_SPARK_USER_JAR, true),
```

Comment:
is it simply that we want everything to be `APP_FILE_PERMISSION` i.e. "world-wide readable and owner writable -> rw-r--r--" per the comment in L681?